translation/message_format.rst

```diff
@@ -178,17 +178,24 @@ you to use literal text in the select statements:
     It's possible to translate ICU MessageFormat messages directly in code,
     without having to define them in any file::
 
+        use Symfony\Component\Translation\MessageCatalogueInterface;
+
         $invitation = '{organizer_gender, select,
             female {{organizer_name} has invited you for her party!}
             male   {{organizer_name} has invited you for his party!}
             other  {{organizer_name} have invited you for their party!}
         }';
 
         // prints "Ryan has invited you for his party!"
-        echo $translator->trans($invitation, [
-            'organizer_name' => 'Ryan',
-            'organizer_gender' => 'male',
-        ]);
+        echo $translator->trans(
+            $invitation,
+            [
+                'organizer_name' => 'Ryan',
+                'organizer_gender' => 'male',
+            ],
+            // Don't forget to suffix the domain with +intl-icu
```

Comment:
```suggestion
            // Appending MessageCatalogueInterface::INTL_DOMAIN_SUFFIX will add the required suffix “+intl-icu” to the domain
```

IMHO, “Don't forget” looks like I have something to do, but the suffix is already added on the line after, so we can just explain what it will do.

Reply:
Works for me.